### DISCUSSION TITLE
Give Access to an already existing Usertable (mysql)

### DISCUSSION THREAD

**Discussion by steffen-mzk** [Budibase/budibase#4412] [Help] [answered]
Hey there! 
Love budibase so much so far. But one thing is very important for me:
I already have an existing Usertable. I want that every user in this table can access budibase apps with there already stored email an password. 
Is this possible?

**Comment by mjashanks** [comment, marked as answer]
Thanks!

This is not possible unless your other system support OpenID connect. 

Your passwords are likely based in your database. There's no way for Budibase to understand your database's hashing system.

**Reply by steffen-mzk** [reply]
Thanks for your fast reply. 
Your answer makes absolut sense. the password doesn’t have to be the same. Maybe a suggestion to solve this with a workaround. 
Can we automate the user management? 
Let‘s say i have already a table with my Users from another system. I could run a cron job or a table change to automaticaly start the adding/change/delete User management. Therefore we even could use the logic in the automation to automatic start the onboarding. 
what do you think about this?

**Reply by mjashanks** [reply]
You could potentially do that - yes. But the issue is with logging into Budibase. How can a user login to Budibase if their password is stored in another database, in a way that it does not understand?

**Reply by steffen-mzk** [reply]
No, they need to get a different Password for budibase then. 
But the „Add User“ Process in budibase would start automatically and the user onboarding begins, the user chooses his password for budibase. 
This would help a lot not to manage ever new user manually or with an import.
